docs/docsite/rst/dev_guide/developing_plugins.rst

```diff
@@ -79,7 +80,7 @@ Developing particular plugin types
 Action plugins
 --------------
 
-Action plugins are useful when you want to modify data that is provided to any module, or filter the data that is returned by the module.
+Action plugins modify the data that is provided to any module, or filter the data that is returned by a module.
```

Comment:
I would say jinja2 filters are useful for those cases, not really worth creating action plugins for that.

action plugins are useful when you need 'local' processing on controller, i.e get local file for `copy: src=`.

Reply:
@bcoca does 46fc7aa7616aa273c87bae74560b4996dffc9e52 do a better job of describing the use case for action plugins?

Reply:
yep